.gitignore

```diff
@@ -2,3 +2,4 @@ phpunit.xml
 vendor
 composer.lock
 composer.phar
+.idea/
```

Comment:
This should be reverted (you can ignore that globally on your system or per project).